lang/java/reef-runtime-spark/src/main/java/org/apache/reef/runtime/spark/InputStreamReaderRunnable.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.reef.runtime.spark;
+
+/**
+ * Created by skanjila on 6/30/17.
+ */
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.UnsupportedEncodingException;
+//
+import org.apache.log4j.Logger;
+//
+
+/**
+ * The InputStreamReaderRunnable class captures
+ * the output streams generated by submitting a
+ * Spark job (from a Java code) to a Spark Cluster.
+ *
+ * @author Mahmoud Parsian (mahmoud.parsian@yahoo.com)
```

Comment:
I actually modified the initial code and brought it into reef, the license is Apache 2.0, I will remove the author part here.